tfx_bsl/beam/run_inference.py

```diff
@@ -71,27 +73,26 @@
                                classification_pb2.Classifications]]
 
 
+class OperationType(object):
+  CLASSIFICATION = 'CLASSIFICATION'
+  REGRESSION = 'REGRESSION'
+  PREDICTION = 'PREDICTION'
+  MULTIHEAD = 'MULTIHEAD'
+
+
 @beam.ptransform_fn
 @beam.typehints.with_input_types(Union[tf.train.Example,
                                        tf.train.SequenceExample])
-@beam.typehints.with_output_types(prediction_log_pb2.PredictionLog)
 def RunInference(  # pylint: disable=invalid-name
     examples: beam.pvalue.PCollection,
     inference_endpoint: model_spec_pb2.InferenceEndpoint
 ) -> beam.pvalue.PCollection:
-  """Run batch offline inference with a model.
-
-  Models need to have the required serving signature as mentioned in
-  [Tensorflow Serving](https://www.tensorflow.org/tfx/serving/signature_defs)
-
-  This function will check model signatures first. Then it will load and run
-  model inference in batch.
+  """Run inference with a model.
```

Comment:
Could you add comments about how to use saved_model_spec and model_endpoint_spec?